tests/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION "${CMAKE_VERSION}")
+
+project(SuperTest LANGUAGES CXX)
+
+find_package(CMakePublic CONFIG REQUIRED)
+find_package(HelloWorld CONFIG REQUIRED)
```

Comment:
If `HelloWorld` is set-up correctly (something this unit test is ultimately testing) you shouldn't have to look for `CMakePublic` again.